packages/mgt-components/src/components/mgt-picker/mgt-picker.ts

```diff
@@ -269,4 +277,28 @@ export class MgtPicker extends MgtTemplatedComponent {
   private handleClick(e: MouseEvent, item: any) {
     this.fireCustomEvent('selectionChanged', item, true, false, true);
   }
+
+  /**
+   * Handles getting the fluent option item in the dropdown and fires a custom
+   * event with it when you press Enter or Backspace keys.
+   *
+   * @param {KeyboardEvent} e
+   */
+  private handleComboboxKeydown = (e: KeyboardEvent) => {
+    let value: string;
+    let item: any;
+    const keyName: string = e.key;
+    const comboBox: HTMLElement = e.target as HTMLElement;
+    const fluentOptionEl = comboBox.querySelector('.selected');
+    if (fluentOptionEl) {
+      value = fluentOptionEl.getAttribute('value');
+    }
+
+    if (['Enter', 'Backspace'].includes(keyName)) {
```

Comment:
This component has a very _weird_ build. In my keyboard, `e.key` shows up as `Backspace`. Now, I have to change that to Tab key because the space bar edits the combobox and overriding it for any other functionality renders searches with a space undoable. The Tab can select an option so I have added it in to fire the custom event.

Reply:
Come to think of it shouldn't space key hander be on the fluent-option? 
That way the input doesn't bubble up to the input

Reply:
Yes, it should. However, the fluent-option has no events it's emitting when I put a `keydown` and `keyup` handlers to it. Being a combobox, the text area and the menu are linked. Maybe to allow typing searches? That's why I implemented the events on the fluent-menu and then handle the selected fluent-option by reading in the DOM children it has and filtering the required option.